src/lib/libraries/backdrops.json

```diff
@@ -215,7 +247,7 @@
     },
     {
         "name": "Jurassic ",
-        "md5": "73fd0531a1aaca5dff2d1c67202ff5bd.svg",
+        "md5": "2d59bed41204445ae9dbcb963b8da14f.svg",
```

Comment:
Did you mean to change jurassic? It seems to be off-center now

Reply:
Hmm. I didn't change this. I'll check with @kosiecki